l3kernel/l3keys.dtx

```diff
@@ -216,7 +217,7 @@
 % \end{function}
 %
 % \begin{function}[added = 2011-08-21, updated = 2013-07-10]
-%   {.choices:nn, .choices:Vn, .choices:on, .choices:xn}
+%   {.choices:nn, .choices:Vn}
```

Comment:
Not that I know of a use case, but why removing `.choices:xn` without providing `.choices:en`?

Reply:
I went here for reducing variants if possible, since we don't have all variants for all key properties. No one has used `.choices:xn`, so it seems it's not currently required to have a expanded variant. But it's a low barrier to having one if that is the overall feeling.